### PR TITLE
Add information around built-in namespaces `nebari-git` and `global`

### LINKED ISSUE #80: [DOC] - Add docs regarding `filesystem` and `default` environments name changes
The environment names for `filesystem` and `default` leads to some misconception about what these environments are used for and how they can be managed. We need to create some documentation around what they are, how they’re used, and what they mean.

Also, we will be changing these environment names to:
- `filsystem` -> `nebari-git`
- `default` -> `global`

### PR DESCRIPTION
**Pull request by viniciusdc**
## Reference Issues or PRs

closes #80 

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: docs/docs/explanations/advanced-env-configuration.md [hunk 1]
```diff
@@ -6,26 +6,21 @@ description: Configure environments with conda-store
...
+<!-- TODO: Update the time to create environments based in new conda-store updates -->
+
 Nebari comes with two default environments that are built during deployment:
 
 - `nebari-git-nebari-git-dask` to [run distributed workflows with Dask][using-dask],
 - `nebari-git-nebari-git-dashboard` to [create shareable dashboard][create-dashboards].
 
-:::note
-The double `nebari-git` is a known consequence of using conda-store for environment management.
+The configuration of each environment in Nebari is achieved through a `environment.<filename>` mapping to a conda environment specification. To configure environments, you can add entries to the `nebari_config.yml` file located in the deployment repository, which will then be used by conda-store to create or update the environment during deployment.
```
Comment (pavithraes) [inline docs/docs/explanations/advanced-env-configuration.md:21]:
```suggestion
The configuration of each environment in Nebari is achieved through a `environment.<filename>` mapping to a conda environment specification. To configure environments, you can add entries to the `nebari-config.yml` file located in the deployment repository, which will then be used by conda-store to create or update the environment during deployment.
```

File: docs/docs/explanations/advanced-env-configuration.md [hunk 2]
```diff
@@ -46,15 +41,51 @@ environments:
...
+
+:::note
+By default, the namespace `nebari-git` is used in a standard Nebari deployment. However, you can change the namespace name by modifying the `conda-store.default_namespace` entry in your `nebari-config.yaml` configuration file. Keep in mind that this setting permanently changes the namespace name.
+:::
+
+When you specify an environment in `nebari_config.yml`, it will be made available for all users and services under the `nebari-git` namespace. Conda-store is responsible for creating them upon request from the deployment process.
+
+However, the conda-store permission model restricts user intervention for both namespaces. This means that `nebari-git` environments can only be modified within the Nebari deployments, while global environments can be locally changed via direct interaction with conda activate. However, any changes made to global environments will only be perceived by the user and not propagated to other users.
+
+Note that the `nebari-git` namespace is modified using nebari-config deployments only, and any modifications to the `global` namespace will only be available locally.
```
Comment (pavithraes) [inline docs/docs/explanations/advanced-env-configuration.md:86]:
```suggestion
:::warning
To reiterate, the `nebari-git` namespace is modified using nebari-config deployments only, and any modifications to the `global` namespace will only be available locally.
:::
```

**Review by pavithraes — APPROVED** [review]
@viniciusdc Thanks! This looks good to me, and the suggested edits are super-minor, so I'll approve. :)

**Comment by viniciusdc** [comment]
> @viniciusdc Thanks! This looks good to me, and the suggested edits are super-minor, so I'll approve. :)

All done, thanks @pavithraes !!! (I need to be more careful when using VSCode replace next time 😓 )

**Pull request merged**